pkg/bmc/virtualmedia_validation_test.go

```diff
@@ -0,0 +1,19 @@
+package bmc
+
+import (
+	"testing"
+
+	"github.com/metal3-io/baremetal-operator/pkg/provisioner/ironic/testserver"
+)
+
+func TestValidateVirtualMedia(t *testing.T) {
+	redfishVirtualMedia := testserver.NewRedfishVirtualMedia(t).Ready()
+	redfishVirtualMedia.Start()
+	defer redfishVirtualMedia.Stop()
+
+	validation := ValidateVirtualMedia(redfishVirtualMedia.Endpoint(), "bmcuser", "bmcpassword", true)
```

Comment:
An additional parameter to pass the client is required here, ie `redfishVirtualMedia.Client()`

Reply:
I've updated the `ValidateVirtualMedia` to receive an `*http.Client` (doing this I've moved the creation of `http.Client` before calling the function in each bmc.